### ISSUE TITLE
Uploaded images not saving

### ISSUE THREAD

**Issue by roberttolton** [froala/Craft-3-Froala-WYSIWYG#19]
I'm trialing this plugin, thus I haven't activated a full license yet. However as I understand it, full functionality should still be available.

This is my field configuration:

![Screenshot 2020-03-16 at 15 41 23](https://user-images.githubusercontent.com/3510875/76775477-50b83f80-679d-11ea-8c3a-589512bef267.png)

I can see that the folders get created in my Froala Content Asset Volume, but no images are saved.

Screen recording here: https://cloud.roberttolton.com/o0uDGLLW

You can see after the page refreshes (doing a save-and-continue), the image is gone.

If i disable the HTML Purifier, then some HTML for the images is saved - however the source of the image is a BLOB URL.

**Comment by roberttolton** [comment]
I can also see this error in my web console when editing the entry:
```
Uncaught TypeError: Cannot read property 'DEFAULTS' of undefined
```

Which is for this:
https://github.com/froala/Craft-3-Froala-WYSIWYG/blob/a5b5571ef724440a0463818b8d8a22b8bdbc5ad8/src/assets/field/dist/js/plugins/craft.js#L28

**Comment by FreekVR** [comment]
Yes I'm having the same issue, I'll take a look at a quick fork to see if I can fix anything re. this :)

**Comment by FreekVR** [comment]
As far as I've been able to reproduce the Craft plugins for the editor itself ,including media asset selection and selection Entries from the link plugin, have never been updated to support Froala v3 and v4 respectively, while this repo has been receiving major version bumps for that package.

Incredibly sloppy in my opinion, especially for a paid plugin.

I will be creating a new ticket for this, as well as sending a message to Froala support, before buying a license.
